src/main/java/org/sonarlint/intellij/actions/ResolveIssueAction.kt

```diff
@@ -0,0 +1,93 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015-2023 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.actions
+
+import com.intellij.codeInsight.intention.PriorityAction
+import com.intellij.icons.AllIcons
+import com.intellij.notification.NotificationGroup
+import com.intellij.notification.NotificationGroupManager
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.DataKey
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.Iconable
+import org.sonarlint.intellij.analysis.AnalysisStatus
+import org.sonarlint.intellij.common.util.SonarLintUtils
+import org.sonarlint.intellij.config.global.ServerConnection
+import org.sonarlint.intellij.core.ProjectBindingManager
+import org.sonarlint.intellij.finding.issue.LiveIssue
+import org.sonarlint.intellij.finding.issue.vulnerabilities.LocalTaintVulnerability
+
+class ResolveIssueAction(private var serverFindingKey: String? = null, private var status: String? = null) :
+    AbstractSonarAction(
+        "Mark Issue As Resolved", "Mark Issue As Resolved", null
+    ), PriorityAction, Iconable {
+
+    companion object {
+        val GROUP: NotificationGroup = NotificationGroupManager.getInstance().getNotificationGroup("SonarLint: Resolve Issue")
+        val ISSUE_KEY = DataKey.create<LiveIssue>("sonarlint_issue")
+        val TAINT_VULNERABILITY_DATA_KEY = DataKey.create<LocalTaintVulnerability>("sonarlint_taint_vulnerability")
```

Comment:
I think those keys are defined somewhere else, and I'm not sure you can re-define them